dstopic/src/lib.rs

```diff
@@ -1 +1,32 @@
-pub mod parse_args;
+use dstopic_command::Command;
+use structopt::StructOpt;
+
+use cmd_config::ConfigCmd;
+use cmd_hello::HelloCmd;
+
+#[derive(Debug, StructOpt)]
+#[structopt(
+    author = "Kat Marchán <kzm@zkat.tech>",
+    about = "Manage your Entropic packages."
+)]
+pub enum Dstopic {
+    #[structopt(about = "Say hello to something")]
+    Hello(HelloCmd),
+    #[structopt(about = "Configuration subcommands.")]
+    Config(ConfigCmd),
+}
+
+impl Command for Dstopic {
+    fn execute(self) -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
```

Comment:
Instead of waiting for clap to grow support for external arguments, would it be worthwhile to load config here and pass it to `cmd.execute()` as a param? (Metaquestion: do we want to replicate the npm cli's ability to overlay config from multiple sources (env, args, config files)? To what degree?)

Reply:
So, I didn't put this into a PR because it was more an experiment, but we do have [a layered config loader](https://github.com/entropic-dev/ds/blob/latest/dstopic/packages/ds-config/src/lib.rs) I hacked together on top of [`config`](https://crates.io/crates/config), but I need to spend a bit of time seeing what might be a good way of gluing that together with the stuff we get from `clap`. I'd rather not be trying to fetch things from two different locations constantly and just have a single state instance that happens to have layered values correctly. I'll give this a whirl in this PR and see what it ends up looking like.

Reply:
Alright, I've got this working. It's not the pretty macrology I was hoping for, but I don't feel like that's necessary for a project-internal thing (as opposed to a nice library I'd publish). I say we live with the ugly, because it's not even that bad.